index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
```

Comment:
```suggestion
```

This is a confusion between payload and securing mechanism processing. this should talk only about the payload which is JSON-LD.

Reply:
I have accepted @mprorock's suggestions, does that work for you, @Sakurann? If not, please suggest some language.